src/main/webapp/script.js

```diff
@@ -133,11 +155,63 @@ function getTaskInfo() {
 }
 
 /**
+ * When the User Profile Page loads, get user info.
+   If no userID is provided in the URL,
+ * default values will be shown.
+ */
+function getUserInfo() {
+  const params = new URLSearchParams(location.search);
+  const userID = params.get('userID');
+  const users = JSON.parse(gJSONusers);
+  for (user in users) {
+    if (users[user].userID == userID) {
+      const title = document.getElementById('user-name-container');
+      title.innerHTML = '<h1>' + users[user].name + '</h1>';
+      const major = document.getElementById('user-major-container');
+      major.innerText = 'Major: ' + users[user].major;
+      const year = document.getElementById('user-year-container');
+      year.innerText = 'Class Year: ' + users[user].year;
+      const numTaskCompleted =
+        document.getElementById('user-num-complete-container');
+      numTaskCompleted.innerText =
+        'Total Tasks Completed: ' + users[user].numTaskCompleted;
+      const prioritySkills =
+        document.getElementById('user-prskills-container');
+      pskills = '';
+      for (skill of users[user].skills) {
+        if (skill.priority == true) {
+          if (pskills == '') {
+            pskills = skill.skill;
+          } else {
+            pskills = (pskills + ', ' + skill.skill);
+          }
+        }
+      }
+      prioritySkills.innerText = 'Priority Skills: ' + pskills;
+      const skills = document.getElementById('user-skills-container');
+      skillString = '';
+      for (skill of users[user].skills) {
+        if (skillString == '') {
+          skillString = skill.skill;
+        } else {
+          skillString = skillString + ', ' + skill.skill;
+        }
+      }
+      skills.innerText = skillString;
+      break;
+    }
+  }
+}
+
+/**
+ * Build ul element for subtasks on Task Page.
  * Build return to project button.
  * @param {Hashmap} task Array of taskIDs.
  * @return {Element} HTML ul element containing a list of tasks.
+ * @param {Array} subtasks Array of taskIDs.
+ * @return {Element} HTML ul element containing a list of subtasks.
  */
-function getProjectReturn(task) {
+function getProjectReturn(task, subtasks) {
```

Comment:
Why is subtasks added?

Reply:
For some reason javascript lint was getting mad when it wasn't because it was defined as a param ... no worries if you change it back I just wanted to pass lint tests for this PR   :)